tests/ip/test_ip_packet.py

```diff
@@ -334,22 +334,14 @@ def test_forward_ip_packet_recomputed_0xffff_chksum(self, duthost, ptfadapter, c
         # THEN DUT recompute new checksum correctly and forward packet as expected.
 
         (peer_ip_ifaces_pair, rif_rx_ifaces, rif_support, ptf_port_idx, pc_ports_map, ptf_indices) = common_param
-        pkt = testutils.simple_ip_packet(
-            eth_dst=duthost.facts["router_mac"],
-            eth_src=ptfadapter.dataplane.get_mac(0, ptf_port_idx),
-            pktlen=1246,
-            ip_src="10.250.40.40",
-            ip_dst="10.156.190.188",
-            ip_proto=47,
-            ip_tos=0x84,
-            ip_id=0,
-            ip_ihl=5,
-            ip_ttl=122,
-        )
-        pkt.payload.flags = 2
+        ip_layer_str = "4500005ecd064000802fffff646b2e0c646b36880000000000000000000000000000000000000000000000000000"
```

Comment:
Please decode ip_layer_str and keep using simple_ip_packet function to construct a packet, the following code relies on the ip_dst to know the output interface, and check counters accordingly.